exec/java-exec/src/test/java/org/apache/drill/exec/store/json/TestJsonRecordReader.java

```diff
@@ -116,6 +117,7 @@ public void testMixedNumberTypes() throws Exception {
           .jsonBaselineFile("jsoninput/mixed_number_types.json")
           .build().run();
     } catch (Exception ex) {
+      ex.printStackTrace();
```

Comment:
not sure why this printStackTrace was added in a different test from the ones that you added...